### ISSUE TITLE
Update Docs/Readme on "emmet1" to indicate deprecation and existence of emmet2

### ISSUE THREAD

**Issue by danieldjewell** [sergeche/emmet-sublime#723]
Per @sergeche in emmetio/pyv8-binaries#35 PyV8 is deprecated (and from reading it, so is emmet-sublime - e.g. "emmet1"?) 

Seeing as there are *many* issues with this and the only "authoritative" information I found about this was @sergeche's comment mentioned previously, would it be possible to maybe update README.md to include a new section in the header indicating this? I think it would help answer a bunch of questions :) 

I was thinking somewhere above the first heading "How to Install"... maybe something like: 

```markdown

## IMPORTANT! This version is being deprecated

This version of emmet ("emmet1") is being in favor of a newer rewritten version that is 100% pure python without the need for PyV8.

***If you are having issues with PyV8*** (e.g. unable to install on Windows 10, errors on install) please try the new version of Emmet2. The PyV8 binaries are deprecated. 

See: https://github.com/emmetio/sublime-text-plugin

```
https://github.com/sergeche/emmet-sublime/blob/51c81af24c8078827f162ff7c0f0a28cd8d5263d/README.md#L8-L15

## Note to @sergeche 
I was going to respond back to your comment in the other repo, but I realized a bunch of other people were having problems installing PyV8 (just like I was just today 😜) and that it might be helpful for everyone else to open the issue here.

**Comment by sergeche** [comment]
Yes, I’ll add deprecation notice when new plugin will be ready. For now, committing to this repo will cause updates for millions of users and will break hacks they made in order to make Emmet work as expected 🙈
